lib/clangimport.cpp

```diff
@@ -775,7 +775,7 @@ Token *clangimport::AstNode::createTokens(TokenList *tokenList)
         return assign;
     }
     if (nodeType == CompoundStmt) {
-        for (AstNodePtr child: children) {
+        for (const AstNodePtr& child: children) {
```

Comment:
I don't feel sure that smart pointers should be passed by const reference. the copy is not very expensive.

Reply:
But it is an overhead and thus valid. If this would be in a very hot path it would cause a performance impact. This the same as the `const QString&` as parameter case.